2024-04-24 19:45:42.584212: W tensorflow/core/common_runtime/type_inference.cc:339] Type inference failed. This indicates an invalid graph that escaped type checking. Error message: INVALID_ARGUMENT: expected compatible input types, but input 1:
type_id: TFT_OPTIONAL
args {
  type_id: TFT_PRODUCT
  args {
    type_id: TFT_TENSOR
    args {
      type_id: TFT_INT32
    }
  }
}
 is neither a subtype nor a supertype of the combined inputs preceding it:
type_id: TFT_OPTIONAL
args {
  type_id: TFT_PRODUCT
  args {
    type_id: TFT_TENSOR
    args {
      type_id: TFT_FLOAT
    }
  }
}
	for Tuple type infernce function 0
	while inferring type of node 'cond_36/output/_23'
2024-04-24 19:45:43.297828: I external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:454] Loaded cuDNN version 8907
2024-04-24 19:45:43.333212: I external/local_xla/xla/service/service.cc:168] XLA service 0x7efb13b667c0 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2024-04-24 19:45:43.333256: I external/local_xla/xla/service/service.cc:176]   StreamExecutor device (0): NVIDIA GeForce RTX 4070 Laptop GPU, Compute Capability 8.9
2024-04-24 19:45:43.336658: I tensorflow/compiler/mlir/tensorflow/utils/dump_mlir_util.cc:269] disabling MLIR crash reproducer, set env var `MLIR_CRASH_REPRODUCER_DIRECTORY` to enable.
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1713980743.407778  204680 device_compiler.h:186] Compiled cluster using XLA!  This line is logged at most once for the lifetime of the process.



























































































































































771/910 [========================>.....] - ETA: 56s - loss: 4.4108
772/910 [========================>.....] - ETA: 55s - loss: 4.4097Create sweep with ID: bfdm67jh
Sweep URL: https://wandb.ai/robert-jacobs/Robert%27s%20Project/sweeps/bfdm67jh
